tests/Unit/Services/CampaignServiceTest.php

```diff
@@ -12,94 +12,6 @@
 
 class CampaignServiceTest extends TestCase
 {
-    /**
-     * A test finding a single campaign.
-     *s
-     * @return void
-     */
-    public function testFindingASingleCampaign()
-    {
-        // Create a test campaign to find.
-        $testCampaign = [
-            'id' => 34,
-            'title' => 'Welcome Home',
-        ];
-
-        // Mock the call to get the campaign from Ashes.
-        $this->mock(Ashes::class)
-            ->shouldReceive('getCampaign')
-            ->once()
-            ->with($testCampaign['id'])
-            ->andReturn([
-                'data' => $testCampaign,
-            ]);
-
-        $campaignService = $this->app->make(CampaignService::class);
-        $cache = new CacheRepository('campaign');
-
-        // Test that the campaign is not in cache.
-        $campaign = $cache->retrieve($testCampaign['id']);
-        $this->assertNull($campaign);
-
-        // If it's not in cache we should make the call to phoenix
-        // and make sure we get back the right campaign and store it in cache
-        // for future requests.
-        $campaign = $campaignService->find($testCampaign['id']);
-        $this->assertEquals($campaign, $testCampaign);
-        $this->assertEquals($cache->retrieve($testCampaign['id']), $testCampaign);
-    }
-
-    /**
-     * A test finding a group of campaigns.
-     *
-     * @test
-     */
-    public function testFindingMultipleCampaigns()
-    {
-        // @TODO: Doesn't work with in-memory cache, which is used in tests.
-        $this->markTestIncomplete();
-
-        $testIds = [57, 4];
-
-        $testCampaigns = [
-            [
-                'id' => 57,
-                'title' => 'Babysitters Club',
-            ],
-            [
-                'id' => 4,
-                'title' => 'Banned Books Club',
-            ],
-        ];
-
-        // Mock the call to get the campaign from Ashes.
-        $this->mock(Ashes::class)
-            ->shouldReceive('getAllCampaigns')
-            ->once()
-            ->andReturn(['data' => $testCampaigns]);
-
-        $campaignService = $this->app->make(CampaignService::class);
-        $cache = new CacheRepository('campaign');
-
-        // Test that the campaign is not in cache.
-        $campaigns = $cache->retrieveMany($testIds);
-        $this->assertNull($campaigns);
-
-        // If it's not in cache we should make the call to Ashes
-        // and make sure we get back the right campaign and store it in cache
-        // for future requests.
-        $campaigns = $campaignService->findAll($testIds);
-        $cachedCampaigns = $cache->retrieveMany($testIds);
-
-        foreach ($testCampaigns as $key => $test) {
-            // Make sure the campaign was returned from Ashes.
-            $this->assertEquals($test, $campaigns[$key]);
-
-            // Make sure the campaigns got stored in cached.
-            $this->assertEquals($test, $cachedCampaigns[$test['id']]);
-        }
-    }
-
```

Comment:
I guess we don't really need tests for this part now since we are literally just pulling straight from the Rogue database?

Reply:
Yeah, that's what I was thinking. 💃